.gitignore

```diff
@@ -0,0 +1,5 @@
+npm-debug.log*
+node_modules
+/build/
+/public/app.css
```

Comment:
Are we sure we want to remove this from git ?

Reply:
I don't know how it get added to `.gitignore`, in fact we don't have that file, removed it from `.gitignore`